src/views/ConsentContinuationOut.vue

```diff
@@ -945,5 +945,19 @@ h2 {
   .value.certifiedby {
     padding-left: 3rem;
   }
+
+  .certify-statement-section {
+    padding-left: 15rem;
```

Comment:
No, don't do this. Use the grid system like the other sections (and/or v-rows/v-cols).